### PR TITLE
DYN-6965 Do not show No Documentation message when In Depth documentation is not available for a node

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

https://jira.autodesk.com/browse/DYN-6965

The PR removes the `No documentation provided` message, when no In Depth documentation is found.

Without documentation:
Screenshot 2024-05-29 at 7 05 42 PM

With documentation:
Screenshot 2024-05-29 at 7 06 08 PM

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

- Do not show No Documentation message when In Depth documentation is not available for a node

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

File: src/DocumentationBrowserViewExtension/MarkdownHandler.cs [hunk 1]
```diff
@@ -91,13 +91,9 @@ internal string ParseToHtml(string nodeNamespace, string packageName)
         {
             var mdFilePath = PackageDocumentationManager.Instance.GetAnnotationDoc(nodeNamespace, packageName);
 
-            string mdString;
+            string mdString = string.Empty;
 
-            if (string.IsNullOrWhiteSpace(mdFilePath) ||
-                !File.Exists(mdFilePath))
-                mdString = ResourceUtilities.LoadContentFromResources(NODE_ANNOTATION_NOT_FOUND, GetType().Assembly);
```
[the remark this answers is not in the document]
Comment (zeusongit) [reply src/DocumentationBrowserViewExtension/MarkdownHandler.cs:98]: Do you mean where else do we use `NODE_ANNOTATION_NOT_FOUND `?
It is also used here https://github.com/DynamoDS/Dynamo/blob/0b23a2e98f9d62945ee2d44042fda8997cd3a775/src/DocumentationBrowserViewExtension/MarkdownHandler.cs#L57
but that method is not referenced anywhere else.

[the remark this answers is not in the document]
Comment (QilongTang) [reply src/DocumentationBrowserViewExtension/MarkdownHandler.cs:98]: I am curious then if we can remove the resource as well

[the remark this answers is not in the document]
Comment (zeusongit) [reply src/DocumentationBrowserViewExtension/MarkdownHandler.cs:98]: Sure, we can always add it back

[the remark this answers is not in the document]
Comment (zeusongit) [reply src/DocumentationBrowserViewExtension/MarkdownHandler.cs:98]: Do we want to also remove the unused method?

[the remark this answers is not in the document]
Comment (QilongTang) [reply src/DocumentationBrowserViewExtension/MarkdownHandler.cs:98]: Sure

**Review by QilongTang — APPROVED** [review]

**Pull request merged**